frontend/packages/kubevirt-plugin/integration-tests/tests/models/virtualMachineTemplate.ts

```diff
@@ -82,8 +98,68 @@ export class VirtualMachineTemplate extends KubevirtUIResource {
     await wizard.waitForCreation();
 
     // Verify VM template is created
+    await this.navigateToDetail();
+  }
+
+  async createVM({
```

Comment:
do we really want this to be duplicated. Can we move this to BaseVirtualMachine ?

Reply:
Good point. I have changed the VM Wizard creating login a bit.
In this commit, I moved it to the Wizard class, which seems to be more logical than to have VirtualMachine  responsible for the wizard logic.
With this separation, it's also less likely to make cyclic dependency.

Reply:
nice refactoring :tada: